packages/app/template/components/nuxt-loading.vue

```diff
@@ -1,115 +1,177 @@
-<template>
-  <div
-    class="nuxt-progress"
-    :style="{
-      'width': percent + '%',
-      'height': height,
-      'background-color': canSuccess ? color : failedColor,
-      'opacity': show ? 1 : 0
-    }"
-  />
-</template>
-
 <script>
-import Vue from 'vue'
-
 export default {
   name: 'nuxt-loading',
   data() {
     return {
       percent: 0,
       show: false,
-      canSuccess: true,
+      canSucceed: true,
+      reversed: false,
+      skipTimerCount: 0,
+      rtl: <%= loading.rtl || false %>,
       throttle: <%= loading.throttle || 200 %>,
-      duration: <%= loading.duration || 1000 %>,
-      height: '<%= loading.height %>',
-      color: '<%= loading.color %>',
-      failedColor: '<%= loading.failedColor %>'
+      duration: <%= loading.duration || 3000 %>,
+      continuous: <%= loading.continuous || false %>
```

Comment:
Do we need the default fallback here? I think we already have them in the config. Might be worth a refactor while the PR is up :thinking:

Reply:
This is a quick fix / workaround to prevent lint errors with `lint:app` in the with-config fixture. If you provide your own loading component, nuxt-loading.vue is still being generated by the builder but without any loading properties as `loading: false`.